examples/rust/image-search/src/main.rs

```diff
@@ -9,7 +9,6 @@ use ahnlich_types::{
     ai::{AIModel, AIServerResponse, PreprocessAction},
     keyval::{StoreInput, StoreName},
     metadata::{MetadataKey, MetadataValue},
-    similarity::Algorithm,
```

Comment:
Could we also make the default processing to be ModelProcessing?

Reply:
Sure why not